test/vars/UtilSpec.groovy

```diff
@@ -423,4 +423,17 @@ class UtilSpec extends JenkinsPipelineSpecification {
         then:
         true
     }
+    def "[util.groovy] getNextSnapshot"() {
+        when:
+        def snapshotVersion = groovyScript.getNextSnapshot('0.12.0')
+        then:
+        snapshotVersion == '0.12.1-SNAPSHOT'
+        
+    }
+    def "[util.groovy] getNextSnapshot"() {
```

Comment:
You can't have 2 test methods with the same name.

Reply:
Please let me know a convenient name for the same:)